src/link/styles.scss

```diff
@@ -45,7 +52,11 @@
   }
 
   @include focus-visible.when-visible {
-    @include styles.link-focus;
+    @include styles.focus-highlight(
```

Comment:
does this cause a regression for links in classic? (because of missing these lines: https://github.com/cloudscape-design/components/blob/main/src/internal/styles/forms/mixins.scss#L16-L19)

can you run through a dev pipeline to check?